code/style.css

```diff
@@ -1,58 +1,169 @@
+header {
+  font-family: "Candal";
+  font-size: 75px;
+  color: #ffffff;
+  text-align: center;
+  margin: 10px;
+  padding: 10px;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: "didot";
+  font-size: 15px;
+  background-image: url(./assets/mystery.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`url("./assets/mystery.jpg");`